@navikt/ds-react/alert/src/Alert.tsx

```diff
@@ -8,7 +8,7 @@ import {
 } from "@navikt/ds-icons";
 import "@navikt/ds-css/alert/index.css";
 
-export interface AlertProps {
+export interface AlertProps extends React.HTMLAttributes<HTMLDivElement> {
   variant: "error" | "warning" | "info" | "success";
   children: React.ReactNode;
   className?: string;
```

Comment:
Vi trenger faktisk ikke definere className når vi arver fra `React.HTMLAttributes`, så denne linja kan fjernes. Det gjelder sikkert andre komponenter også.

Reply:
React docgen plukker ikke opp extended props riktig, så vi må sette className for at proptablet skal være riktig.